tests/performance/test_performance.py

```diff
@@ -54,15 +63,177 @@ def test_performance(config_path):
     with open(config_path, 'r') as config_file:
         config = json.load(config_file)
 
-    transformer = get_instance(config['transformer'], **config['kwargs'])
-    dataset_gen = get_instance(config['dataset'])
+    transformer_instance = get_instance(config['transformer'], **config['kwargs'])
+    dataset_generator = get_instance(config['dataset'])
 
-    out = profile_transformer(transformer, dataset_gen, config['transform_size'],
-                              config['fit_size'])
+    out = profile_transformer(
+        transformer=transformer_instance,
+        dataset_generator=dataset_generator,
+        fit_size=config['fit_size'],
+        transform_size=config['transform_size'],
+    )
 
     assert out['Fit Time'] < config['expected']['fit']['time']
     assert out['Fit Memory'] < config['expected']['fit']['memory']
     assert out['Transform Time'] < config['expected']['transform']['time']
     assert out['Transform Memory'] < config['expected']['transform']['memory']
     assert out['Reverse Transform Time'] < config['expected']['reverse_transform']['time']
     assert out['Reverse Transform Memory'] < config['expected']['reverse_transform']['memory']
+
+
+def _round_to_magnitude(value):
+    for digits in range(-15, 15):
```

Comment:
I'm wondering if this range can be problematic, if the metric is large in value then I'm questioning this example:
```
np.round(123, -2) # 100
np.round(123, -1) # 120
```
which case is the desired case?

Reply:
I believe he's trying to round it to the largest possible tens place, so the former would be preferred.

Reply:
That makes sense, I forgot about the multiplication by 10 part.